### ISSUE TITLE
Blurry Text and Images

### ISSUE THREAD

**Issue by AndrewCurrie** [rubymotion-community/Joybox#20]
Both labels and sprites are rendering very blurry. I have the proper -hd suffixes setup and still the same result. Even if the there was a problem with the sprite rendering, the labels should still be fully crisp in retina mode right? Is this a known issues or am I just setting something up incorrectly?

Here is an example of the text issue I am talking about:
![screen shot 2013-05-20 at 11 40 35 am](https://f.cloud.github.com/assets/1109316/536938/d3acc164-c163-11e2-865f-02b61aed2b08.png)

**Comment by NSCoder** [comment]
Hey Andrew!

Thanks for the report! Actually is a bug on the Director configuration, it will be fixed in the next release but you can add the following line in your app delegate after the setup initialization:

``` ruby
# After the following
@director = Joybox::Configuration.setup do
  director display_stats: true
end

# Add this line
@director.enableRetinaDisplay(true)
```

Hope it helps, thank you!

JK

**Comment by AndrewCurrie** [comment]
Works like a charm! Thanks

**Comment by nside** [comment]
I just hit the same issue. I suggest keeping this open until the fix is released.

**Comment by NSCoder** [comment]
Agree, lets keep it open! :)

**Comment by NSCoder** [comment]
Hey guys!

Now is fixed in 1.1.0 version, thanks for waiting :)
